builder/coordinator.go

```diff
@@ -265,3 +227,73 @@ func (b *Builder) visit(n *Node) {
 
 	b.BuildQueue <- n
 }
+
+func install(job *Node) error {
+	buildOut := ""
+	if build.Getenv("BUILD_OUT") != "" {
+		buildOut = build.Getenv("BUILD_OUT")
+	} else {
+		buildOut = filepath.Join(
+			util.GetProjectPath(),
+			"build_out",
+		)
+	}
```

Comment:
If this is variable, it probably shouldn't be hard-coded in the `util.Hash` functions.